airbyte-webapp/src/packages/cloud/services/thirdParty/launchdarkly/LDExperimentService.tsx

```diff
@@ -164,7 +140,9 @@ const LDInitializationWrapper: React.FC<React.PropsWithChildren<{ apiKey: string
 
   // Whenever the user, locale or workspaceId changes, we need to re-identify with launchdarkly
   useEffect(() => {
-    ldClient.current?.identify(mapUserToLDUser(user, locale, workspaceId || null));
+    const userContext = createUserContext(user, locale);
+    const contexts = workspaceId ? createMultiContext(userContext, createWorkspaceContext(workspaceId)) : userContext;
```

Comment:
Nitpick: for easier reading: shouldn't we just always wrap it into a multi context even if it's only the user context in there, so we just sometimes fill it with user and workspace and sometimes just with workspace?

Reply:
So according to the LD docs, a single context should be declared without the "multi" wrapper. But I just tested it and seemingly a multi-context containing only a single context works fine. Happy to change that if you think it increases readability.